### PR TITLE
Ray Logging

### PR DESCRIPTION
**Pull request by malachi-constant**
### Feature 
- Add RayLogger class to instantiate logger object per Ray Task.

### Relates
- #1551 

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

**Comment by malachi-constant** [comment]
### AWS CodeBuild CI Report

* CodeBuild project: GitHubDistributedCodeBuild6-jWcl5DLmvupS
* Commit ID: 47a3225d7d7be3f4f9b7417ff5b96a5f48c16502
* Result: FAILED
* [Build Logs](https://p0rshq9q5i.execute-api.us-east-1.amazonaws.com/Prod/buildlogs?key=f7a47d23-1d25-4482-8b00-9bb29f51e987%2Fbuild.log) (available for 30 days)

*Powered by [github-codebuild-logs](https://serverlessrepo.aws.amazon.com/applications/arn:aws:serverlessrepo:us-east-1:277187709615:applications~github-codebuild-logs), available on the [AWS Serverless Application Repository](https://aws.amazon.com/serverless/serverlessrepo/)*

**Comment by malachi-constant** [comment]
### AWS CodeBuild CI Report

* CodeBuild project: GitHubLoadTests5656BB24-s6u9F3qN9oFy
* Commit ID: 47a3225d7d7be3f4f9b7417ff5b96a5f48c16502
* Result: SUCCEEDED
* [Build Logs](https://pciyom2pbh.execute-api.us-east-1.amazonaws.com/Prod/buildlogs?key=f11d9c34-05ef-4934-90d8-46c1c745fc0a%2Fbuild.log) (available for 30 days)

*Powered by [github-codebuild-logs](https://serverlessrepo.aws.amazon.com/applications/arn:aws:serverlessrepo:us-east-1:277187709615:applications~github-codebuild-logs), available on the [AWS Serverless Application Repository](https://aws.amazon.com/serverless/serverlessrepo/)*

**Comment by malachi-constant** [comment]
### AWS CodeBuild CI Report

* CodeBuild project: GitHubStandardCodeBuild8C06-llutOAimTATs
* Commit ID: 47a3225d7d7be3f4f9b7417ff5b96a5f48c16502
* Result: SUCCEEDED
* [Build Logs](https://jsv9u8gote.execute-api.us-east-1.amazonaws.com/Prod/buildlogs?key=d0968b8b-551d-4d42-867c-a41bc7405950%2Fbuild.log) (available for 30 days)

*Powered by [github-codebuild-logs](https://serverlessrepo.aws.amazon.com/applications/arn:aws:serverlessrepo:us-east-1:277187709615:applications~github-codebuild-logs), available on the [AWS Serverless Application Repository](https://aws.amazon.com/serverless/serverlessrepo/)*

**Comment by malachi-constant** [comment]
### AWS CodeBuild CI Report

* CodeBuild project: GitHubDistributedCodeBuild6-jWcl5DLmvupS
* Commit ID: 29e95aa41637f199b63a402b062cf4f460f7f3d1
* Result: FAILED
* [Build Logs](https://p0rshq9q5i.execute-api.us-east-1.amazonaws.com/Prod/buildlogs?key=c940623f-6387-4f5b-99ca-abfa090c80c5%2Fbuild.log) (available for 30 days)

*Powered by [github-codebuild-logs](https://serverlessrepo.aws.amazon.com/applications/arn:aws:serverlessrepo:us-east-1:277187709615:applications~github-codebuild-logs), available on the [AWS Serverless Application Repository](https://aws.amazon.com/serverless/serverlessrepo/)*

**Comment by malachi-constant** [comment]
### AWS CodeBuild CI Report

* CodeBuild project: GitHubDistributedCodeBuild6-jWcl5DLmvupS
* Commit ID: d63d79dce7b85a46e63cd58c63c0f108beba1522
* Result: FAILED
* [Build Logs](https://p0rshq9q5i.execute-api.us-east-1.amazonaws.com/Prod/buildlogs?key=d20b68ea-9719-4ef2-9557-61b78a8ae6eb%2Fbuild.log) (available for 30 days)

*Powered by [github-codebuild-logs](https://serverlessrepo.aws.amazon.com/applications/arn:aws:serverlessrepo:us-east-1:277187709615:applications~github-codebuild-logs), available on the [AWS Serverless Application Repository](https://aws.amazon.com/serverless/serverlessrepo/)*

**Comment by malachi-constant** [comment]
### AWS CodeBuild CI Report

* CodeBuild project: GitHubStandardCodeBuild8C06-llutOAimTATs
* Commit ID: 29e95aa41637f199b63a402b062cf4f460f7f3d1
* Result: SUCCEEDED
* [Build Logs](https://jsv9u8gote.execute-api.us-east-1.amazonaws.com/Prod/buildlogs?key=9021983a-76b5-4058-96e2-56012d9106b0%2Fbuild.log) (available for 30 days)

*Powered by [github-codebuild-logs](https://serverlessrepo.aws.amazon.com/applications/arn:aws:serverlessrepo:us-east-1:277187709615:applications~github-codebuild-logs), available on the [AWS Serverless Application Repository](https://aws.amazon.com/serverless/serverlessrepo/)*

**Comment by malachi-constant** [comment]
### AWS CodeBuild CI Report

* CodeBuild project: GitHubLoadTests5656BB24-s6u9F3qN9oFy
* Commit ID: d63d79dce7b85a46e63cd58c63c0f108beba1522
* Result: SUCCEEDED
* [Build Logs](https://pciyom2pbh.execute-api.us-east-1.amazonaws.com/Prod/buildlogs?key=aaae15f3-4d0c-4f73-bc57-bb29caad58bd%2Fbuild.log) (available for 30 days)

*Powered by [github-codebuild-logs](https://serverlessrepo.aws.amazon.com/applications/arn:aws:serverlessrepo:us-east-1:277187709615:applications~github-codebuild-logs), available on the [AWS Serverless Application Repository](https://aws.amazon.com/serverless/serverlessrepo/)*

**Comment by malachi-constant** [comment]
### AWS CodeBuild CI Report

* CodeBuild project: GitHubDistributedCodeBuild6-jWcl5DLmvupS
* Commit ID: 697307a8c3b0c136221eddd2100a1ec88b7fdfef
* Result: FAILED
* [Build Logs](https://p0rshq9q5i.execute-api.us-east-1.amazonaws.com/Prod/buildlogs?key=315669c2-ce2c-4d69-b7d3-6d339688f374%2Fbuild.log) (available for 30 days)

*Powered by [github-codebuild-logs](https://serverlessrepo.aws.amazon.com/applications/arn:aws:serverlessrepo:us-east-1:277187709615:applications~github-codebuild-logs), available on the [AWS Serverless Application Repository](https://aws.amazon.com/serverless/serverlessrepo/)*

**Comment by malachi-constant** [comment]
### AWS CodeBuild CI Report

* CodeBuild project: GitHubLoadTests5656BB24-s6u9F3qN9oFy
* Commit ID: 697307a8c3b0c136221eddd2100a1ec88b7fdfef
* Result: SUCCEEDED
* [Build Logs](https://pciyom2pbh.execute-api.us-east-1.amazonaws.com/Prod/buildlogs?key=2d946eb2-a4ad-483b-987c-774616577a3d%2Fbuild.log) (available for 30 days)

*Powered by [github-codebuild-logs](https://serverlessrepo.aws.amazon.com/applications/arn:aws:serverlessrepo:us-east-1:277187709615:applications~github-codebuild-logs), available on the [AWS Serverless Application Repository](https://aws.amazon.com/serverless/serverlessrepo/)*

**Comment by malachi-constant** [comment]
### AWS CodeBuild CI Report

* CodeBuild project: GitHubDistributedCodeBuild6-jWcl5DLmvupS
* Commit ID: 56504335ec9d2ceaf1d3130205a5c16ea050cd52
* Result: FAILED
* [Build Logs](https://p0rshq9q5i.execute-api.us-east-1.amazonaws.com/Prod/buildlogs?key=5911f933-1419-4d89-acea-d39e6c90a36b%2Fbuild.log) (available for 30 days)

*Powered by [github-codebuild-logs](https://serverlessrepo.aws.amazon.com/applications/arn:aws:serverlessrepo:us-east-1:277187709615:applications~github-codebuild-logs), available on the [AWS Serverless Application Repository](https://aws.amazon.com/serverless/serverlessrepo/)*

**Comment by malachi-constant** [comment]
### AWS CodeBuild CI Report

* CodeBuild project: GitHubStandardCodeBuild8C06-llutOAimTATs
* Commit ID: 697307a8c3b0c136221eddd2100a1ec88b7fdfef
* Result: SUCCEEDED
* [Build Logs](https://jsv9u8gote.execute-api.us-east-1.amazonaws.com/Prod/buildlogs?key=dad3e633-d2a5-4967-b6ff-9f145b40f903%2Fbuild.log) (available for 30 days)

*Powered by [github-codebuild-logs](https://serverlessrepo.aws.amazon.com/applications/arn:aws:serverlessrepo:us-east-1:277187709615:applications~github-codebuild-logs), available on the [AWS Serverless Application Repository](https://aws.amazon.com/serverless/serverlessrepo/)*

**Comment by malachi-constant** [comment]
### AWS CodeBuild CI Report

* CodeBuild project: GitHubLoadTests5656BB24-s6u9F3qN9oFy
* Commit ID: 56504335ec9d2ceaf1d3130205a5c16ea050cd52
* Result: SUCCEEDED
* [Build Logs](https://pciyom2pbh.execute-api.us-east-1.amazonaws.com/Prod/buildlogs?key=3da8d446-d05f-4bdf-a841-ff280384410d%2Fbuild.log) (available for 30 days)

*Powered by [github-codebuild-logs](https://serverlessrepo.aws.amazon.com/applications/arn:aws:serverlessrepo:us-east-1:277187709615:applications~github-codebuild-logs), available on the [AWS Serverless Application Repository](https://aws.amazon.com/serverless/serverlessrepo/)*

File: tests/load/test_s3.py [hunk 1]
```diff
@@ -21,7 +21,7 @@ def df_xl():
 @pytest.mark.repeat(1)
 @pytest.mark.parametrize("benchmark_time", [180])
 def test_s3_select(benchmark_time):
-    path = "s3://ursa-labs-taxi-data/2018/1*.parquet"
+    path = "s3://ursa-labs-taxi-data/2018/10*.parquet"
```
[the remark this answers is not in the document]
Comment (malachi-constant) [reply tests/load/test_s3.py]: local testing made it into the remote 😮‍💨

**Comment by malachi-constant** [comment]
### AWS CodeBuild CI Report

* CodeBuild project: GitHubStandardCodeBuild8C06-llutOAimTATs
* Commit ID: 1c82665310636e208fd94068bccad0d54337ade2
* Result: FAILED
* [Build Logs](https://jsv9u8gote.execute-api.us-east-1.amazonaws.com/Prod/buildlogs?key=dfa1aaec-4f1e-4227-b1d6-775889897931%2Fbuild.log) (available for 30 days)

*Powered by [github-codebuild-logs](https://serverlessrepo.aws.amazon.com/applications/arn:aws:serverlessrepo:us-east-1:277187709615:applications~github-codebuild-logs), available on the [AWS Serverless Application Repository](https://aws.amazon.com/serverless/serverlessrepo/)*

**Comment by malachi-constant** [comment]
### AWS CodeBuild CI Report

* CodeBuild project: GitHubStandardCodeBuild8C06-llutOAimTATs
* Commit ID: 86785b0b793afe280b3db318ae85121cb7efa5c2
* Result: FAILED
* [Build Logs](https://jsv9u8gote.execute-api.us-east-1.amazonaws.com/Prod/buildlogs?key=acc10b7c-413b-43da-9be7-daa1a0596837%2Fbuild.log) (available for 30 days)

*Powered by [github-codebuild-logs](https://serverlessrepo.aws.amazon.com/applications/arn:aws:serverlessrepo:us-east-1:277187709615:applications~github-codebuild-logs), available on the [AWS Serverless Application Repository](https://aws.amazon.com/serverless/serverlessrepo/)*

**Comment by malachi-constant** [comment]
### AWS CodeBuild CI Report

* CodeBuild project: GitHubDistributedCodeBuild6-jWcl5DLmvupS
* Commit ID: 86785b0b793afe280b3db318ae85121cb7efa5c2
* Result: FAILED
* [Build Logs](https://p0rshq9q5i.execute-api.us-east-1.amazonaws.com/Prod/buildlogs?key=429f9cc4-d34d-417b-87e8-4f8d0b1b39d1%2Fbuild.log) (available for 30 days)

*Powered by [github-codebuild-logs](https://serverlessrepo.aws.amazon.com/applications/arn:aws:serverlessrepo:us-east-1:277187709615:applications~github-codebuild-logs), available on the [AWS Serverless Application Repository](https://aws.amazon.com/serverless/serverlessrepo/)*

**Review by jaidisido — APPROVED** [review]

**Pull request merged**